internal/incoming/odfi/processor.go

```diff
@@ -150,9 +152,12 @@ func processFile(path string, auditSaver *AuditSaver, fileProcessors Processors)
 	dir, filename := filepath.Split(path)
 	dir = filepath.Base(dir)
 
+	// Allow an override of root path or use legacy path
+	folder := strx.Or(os.Getenv("RDFI_FOLDER"), "odfi")
```

Comment:
This name confuses me. We're storing files from the ODFI and potentially as the RDFI here. I'd rather move forward with the ODFI/RDFI split than merge this into v0.16.0